### DISCUSSION TITLE
Youtube Music Video Player

### DISCUSSION THREAD

**Discussion by MysticNynja** [music-assistant/support#1045] [Feature requests and ideas]
Is there a way to add music videos from YouTube or any other source?  For my use case it's YouTube.

I'd like to be able to add a video Playlist in music assistant and be able to watch on a selected video source in HA.

**Comment by OzGav** [comment]
Sorry only YouTube music is supported at this time.

**Reply by OzGav** [reply]
While video will never be supported directly one thing you can try to workaround not being able to find a specific track from a video in YTM, is adding the video to a playlist from within Youtube. It should show up in the same playlist inside YTM. However, if the item is shown as 'unplayable' (i.e. a gray exlamation mark !) then there is nothing that can be done to get it to play.

**Reply by stedju499** [reply]
It possible to get something like Youtube plugin in LMS? 
[LMS plugins](https://sourceforge.net/p/lms-plugins-philippe44/)

**Reply by OzGav** [reply]
If you want a particular feature then post a discussion about it. The project isn’t trying to replicate LMS
